package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "katie",
```

Comment:
If you don't have a `package.json` file in your repo,  then anybody else who clones it down won't be able to install the necessary dependencies without combing through the code to find all the external packages you'd required.

Reply:
Also, you can initialize a `package.json` file with `npm init`